.github/workflows/deploy.yml

```
@@ -26,6 +26,7 @@ jobs:
           echo "ALGOLIA_INDEX=${{ secrets.ALGOLIA_INDEX_DEV }}" >> $GITHUB_ENV
           echo "ALGOLIA_API_INDEXING_KEY=${{ secrets.ALGOLIA_API_INDEXING_KEY_DEV }}" >> $GITHUB_ENV
           echo "ALGOLIA_API_SEARCH_KEY=${{ secrets.ALGOLIA_API_SEARCH_KEY_DEV }}" >> $GITHUB_ENV
+          echo "GTM_ID==${{ secrets.GTM_ID_DEV }}" >> $GITHUB_ENV
```

sure you need double `=` sign?
```suggestion
          echo "GTM_ID=${{ secrets.GTM_ID_DEV }}" >> $GITHUB_ENV
```